test/e2e/alertmanager_test.go

```diff
@@ -171,8 +171,6 @@ func testAlertmanagerReady(t *testing.T, name, ns string, validator ...validator
 }
 
 func testAlertmanagerTenancyAPI(t *testing.T, host string) {
-	t.Helper()
```

Comment:
I assume we'll get better context without it?

Reply:
Correct.  Otherwise it's hard to know which step failed exactly.